### ISSUE TITLE
repeat playback between the endpoints of a region

### ISSUE THREAD

**Issue by harrietsCreativeDabblets** [katspaugh/wavesurfer.js#826]
Hi,
So I am familiar with the regions plugin, and that you can define multiple regions on your audio wav graph. 

I would appreciate some help and input into the following:

Is it possible, and any tips on how to:
1. Allow only one region to be defined at any given time?
2. Repeat playback of this one region - that is, when the region is selected, playback that area in the audio, over and over.

Ta,
Harriet

**Comment by katspaugh** [comment]
To replay a region, either call `region.playLoop()` or specify `loop: true` in the region params when creating a region.

As for the single region, you need to listen to the `region-created` event and delete any previous regions.

**Comment by harrietsCreativeDabblets** [comment]
Thanks for your reply. How do I delete any previous regions? I tried clearRegions() which obviously clears everything, even the one I have just created?

**Comment by harrietsCreativeDabblets** [comment]
Hello Again,
Can you please help with a basic demo/example of how to get the region plugin up and running?
I am trying to continuously loop over a region, and at any point in time to only allow for one region, but this seems difficult not having anything to go by?
I am particularly interested in the annotations tutorial, as I thing that eventually I would like to go that way. I cannot get the annotation tutorial up and running, and would really appreciate some help with this.
Thank you kindly, and for your great work on this project.

**Comment by harrietsCreativeDabblets** [comment]
Hello,

I have dug into the code a bit, and found that in the wavesurfer.regions.js file I could update the add function, to delete any existing regions from there, before I add a new region. Works perfectly.

However, when I play, it does not loop over the region that was selected, even though I have loop set to true in the attributes. The only time I saw the app loop through the defined region, is when I preloaded a region in the code, and when the DOM loaded, press play without doing anything else. Any advice on how to get a more consistent looping playback experience?
